src/gwt/src/org/rstudio/studio/client/workbench/views/terminal/TerminalPopupMenu.java

```diff
@@ -44,6 +45,7 @@
 {
    public TerminalPopupMenu(TerminalList terminals)
    {
+      super();
```

Comment:
A Java constructor automatically does a call to no-argument `super()` so this isn't needed.